util/helm/cmd.go

```diff
@@ -221,6 +221,9 @@ func (c *Cmd) template(chartPath string, opts *TemplateOpts) (string, error) {
 	for _, v := range opts.APIVersions {
 		args = append(args, "--api-versions", v)
 	}
+	if c.HelmVer.additionalTempalteArgs != nil {
```

Comment:
I guess there is a typo in the field name, `additionalTempalteArgs` should really be `additionalTemplateArgs`

Reply:
Thank you! Fixed. :)